
diff -Naur a/fmc-bus/kernel/Makefile b/fmc-bus/kernel/Makefile
--- a/fmc-bus/kernel/Makefile     2014-11-08 04:36:30.957186266 +0100
+++ b/fmc-bus/kernel/Makefile     2014-11-08 04:50:12.313185206 +0100
@@ -1,7 +1,7 @@
 # include parent_common.mk for buildsystem's defines
 # It allows you to inherit an environment configuration from larger project
 REPO_PARENT=../..
 -include $(REPO_PARENT)/parent_common.mk
-LINUX ?= /lib/modules/$(shell uname -r)/build
+LINUX ?= $(KERNEL_SRC)
 
 GIT_VERSION = $(shell git describe --dirty --long --tags)
